cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -262,7 +262,7 @@
 
     <property>
       <name>stream.bind.address</name>
-      <value>127.0.0.1</value>
```

Comment:
I think binding on all interfaces is nice as a default, but I think in production, developers should be configuring these properties. At least, that's how I interpreted this issue: https://issues.cask.co/browse/CDAP-1000